Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM antora/antora
+
+LABEL maintainer="matthew@matthewsetter.com"
+
+# Enable Lunr site search integration and generation of a Lunr search index by default 
+ENV DOCSEARCH_ENABLED true
+ENV DOCSEARCH_ENGINE lunr
+
+WORKDIR /antora
+
+COPY package* /antora/
+
+RUN apk add --no-cache build-base libressl-dev libcurl libgit2-dev python && \
```

Comment:
Where are these packages required?

Reply:
I installed them as the build failed without them.